# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/CODE_OF_CONDUCT.md
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/CODE_OF_CONDUCT.md b/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/CODE_OF_CONDUCT.md
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/src/vendor/SPIRV-Cross/CODE_OF_CONDUCT.md
+++ /dev/null
@@ -1,1 +0,0 @@
-A reminder that this issue tracker is managed by the Khronos Group. Interactions here should follow the Khronos Code of Conduct (https://www.khronos.org/developers/code-of-conduct), which prohibits aggressive or derogatory language. Please keep the discussion friendly and civil.